hspy_bundle/NSIS_installer_script.nsi

```diff
@@ -307,17 +307,18 @@ SectionIn RO
 	File /r "${WINPYTHON_PATH}\*"
 	${If} $InstMode = 2
 	; Create right-click context menu entries for Hyperspy Here
-		Exec 'cmd.exe /C ""${APP_INSTDIR}\WinPython Command Prompt.exe" install_hyperspy_here & exit"'
+		Exec 'cmd.exe /C ""${APP_INSTDIR}\WinPython Command Prompt.exe" jupyter_context-menu_add & exit"'
 	Sleep 3000
 	${EndIf}
 
 	${If} $InstMode <> 1
 	; Create StartMenu shortcuts
 		CreateDirectory "$SMPROGRAMS\${APPNAME}"
-		CreateShortCut "$SMPROGRAMS\${APPNAME}\${APPNAME}.lnk" "${APP_INSTDIR}\${PYTHON_FOLDER}\Scripts\hyperspy.bat"
+		CreateShortCut "$SMPROGRAMS\${APPNAME}\WinPython prompt.lnk" "${APP_INSTDIR}\WinPython Command Prompt.exe"
+		CreateShortCut "$SMPROGRAMS\${APPNAME}\IPython.lnk" "${APP_INSTDIR}\${PYTHON_FOLDER}\Scripts\ipython.exe"

```

Comment:
Sounds worrisome, but I suggest to wait until we actually test it
before taking any action.

On Thu, 2016-02-18 at 09:37 -0800, Vidar Tonaas Fauske wrote:
> In hspy_bundle/NSIS_installer_script.nsi:
> >  	Sleep 3000
> >  	${EndIf}
> >  
> >  	${If} $InstMode <> 1
> >  	; Create StartMenu shortcuts
> >  		CreateDirectory "$SMPROGRAMS\${APPNAME}"
> > -		CreateShortCut
> "$SMPROGRAMS\${APPNAME}\${APPNAME}.lnk"
> "${APP_INSTDIR}\${PYTHON_FOLDER}\Scripts\hyperspy.bat"
> > +		CreateShortCut "$SMPROGRAMS\${APPNAME}\WinPython
> prompt.lnk" "${APP_INSTDIR}\WinPython Command Prompt.exe"
> > +		CreateShortCut
> "$SMPROGRAMS\${APPNAME}\IPython.lnk"
> "${APP_INSTDIR}\${PYTHON_FOLDER}\Scripts\ipython.exe"
> So, either we have to patch all .exe files in installation step, or
> we cannot rely on any .exe files created by pip. See also
> pypa/pip#2845.
> —
> Reply to this email directly or view it on GitHub.